ui/app/src/components/workspaces/WorkspaceOperationsPanel.tsx

```diff
@@ -0,0 +1,100 @@
+import { DefaultPalette, IStackItemStyles, IStackStyles, Stack } from "@fluentui/react";
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Operation } from '../../models/operation';
+import { Workspace } from '../../models/workspace';
+import { ApiEndpoint } from '../../models/apiEndpoints';
+import { HttpMethod, ResultType, useAuthApiCall } from '../../useAuthApiCall';
+import config from '../../config.json';
+
+
+
+interface WorkspaceOperationPanelProps {
+    workspace: Workspace
+}
+
+interface WorkspaceOperationPanelItemProps {
+    header: String,
+    val: String
+}
+
+const WorkspaceOperationPanelItem: React.FunctionComponent<WorkspaceOperationPanelItemProps> = (props: WorkspaceOperationPanelItemProps) => {
+
+
+
+    const stackItemStyles: IStackItemStyles = {
+        root: {
+            padding: 5,
+            width: 150,
+            color: DefaultPalette.neutralSecondary
+        }
+    }
+    return(
+        <>
+            <Stack wrap horizontal>
+                <Stack.Item grow styles={stackItemStyles}>
+                    {props.header}
+                </Stack.Item>
+                <Stack.Item grow styles={stackItemStyles}>
+                    : {props.val}
+                </Stack.Item>
+            </Stack>
+        </>
+    );
+}
+
+export const WorkspaceOperationsPanel: React.FunctionComponent<WorkspaceOperationPanelProps> = (props: WorkspaceOperationPanelProps) => {
+    const apiCall = useAuthApiCall();
+    const { workspaceId } = useParams();
+    const [workspaceOperations, setWorkspaceOperations] = useState([] as Array<Operation>)
+    
+
+    const getOperations = async () => {
```

Comment:
I think we should wrap this in the `useEffect()` hook here - see the other components for an example. It's a 'lifecycle' hook that means it will fire as the component loads and then only re-fire if it needs to.

Reply:
Exactly the feedback I wanted! Thanks. adding useEffect for the lifecycle. Also, should we also have a loadingState ?